bootstrap.sh

```diff
@@ -529,13 +537,6 @@ software_update
 SENTRY_ROOT="$CODE_ROOT/sentry"
 GETSENTRY_ROOT="$CODE_ROOT/getsentry"
 
-# TODO add env vars to ~/.sentryrc
-# This DSN is for our bootstrap script, not sure if we want it in profile
-# SENTRY_DSN=https://b70e44882d494c68a78ea1e51c2b17f0@o1.ingest.sentry.io/5480435
-# This will be used to measure webpack
-# SENTRY_INSTRUMENTATION=1
-
-install_sentry_cli
```

Comment:
I think the reason why it's there is because it needs xcode CLT installed

Reply:
https://github.com/getsentry/bootstrap-sentry/pull/16/files#diff-99d5a2a8ca4290b61836b3ad37941f2b925e9ff3151c90dcd5ea82364e4122a1R317-R318